### PR TITLE
Remove links for older versions of kubernetes

### PR DESCRIPTION
**Pull request by ebauman**
Upgrade paths listed on this page (1.7, etc) had broken links. Referred those links to the last working copy, which was in v1.11 docs.

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
> For 1.12 Features: set Milestone to 1.12 and Base Branch to release-1.12
> Help editing and submitting pull requests:  https://deploy-preview-9510--kubernetes-io-master-staging.netlify.com/docs/contribute/start/#submit-a-pull-request.
> Help choosing which branch to use, see
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by ebauman** [comment]
Please check CNCF CLA registration again, @k8s-ci-robot

**Comment by neolit123** [comment]
/check-cla

these pages were removed with the 1.12 release because they are outdated.
i wonder if we should remove them completely.

/assign @timothysc

**Comment by tengqm** [comment]
@ebauman Tend to agree with @neolit123 that the old links can be safely dropped.

**Comment by ebauman** [comment]
I can get behind that idea. 
I added a second commit, which if everyone finds agreeable, does the following:
- Removes older links from this page
- Adds an emphasized line to instruct users of older k8s versions to refer to older documentation sets.

**Comment by ebauman** [comment]
Additionally, not sure what is up with my CNCF CLA confirmation. These commits were made using the github website, which has my primary email configured. Sorry for any issues with this.

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md [hunk 1]
```diff
@@ -19,14 +19,10 @@ cluster if necessary.
...
- * [1.7.x to 1.7.y upgrades](https://v1-11.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-8/)
- * [1.7 to 1.8 upgrades](https://v1-11.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-8/)
- * [1.8.x to 1.8.y upgrades](https://v1-11.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-8/)
- * [1.8.x to 1.9.x upgrades](https://v1-11.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-9/)
- * [1.9.x to 1.9.y upgrades](https://v1-11.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-9/)
- * [1.9.x to 1.9.y HA cluster upgrades](https://v1-11.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-ha/)
  * [1.10 to 1.11 upgrades](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-11/)
  * [1.11 to 1.12 upgrades](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-12/)
+ 
+_For older versions, please refer to older documentation sets on the Kubernetes website._
```
Comment (neolit123) [inline content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md]: @tengqm should we link here instead?

https://v1-11.docs.kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-upgrade/

**Comment by neolit123** [comment]
@ebauman 
if you are still having CLA issues after the bellow command, contact the CNCF help desk.

/check-cla

**Comment by ebauman** [comment]
/check-cla

Once more, I think I got it right this time.

**Comment by neolit123** [comment]
> Once more, I think I got it right this time.

looks good.
waiting on feedback from @tengqm about the new added sentence.

thanks.

File: content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md]: No, I don't think so. Documentation for upgrade is not yet versioned by itself. We are still maintaining them as regular topics.

**Comment by timothysc** [comment]
/approve 

@kubernetes/sig-docs-pr-reviews

**Pull request merged**